src/dispatch/stage.rs

```diff
@@ -156,6 +157,32 @@ impl<'a> StagesBuilder<'a> {
         self.stages
     }
 
+    pub fn print_par_seq(&self, map: &FxHashMap<String, SystemId>) {
+        println!("seq![");
+        for stage in &self.ids {
+            println!("\tpar![");
+            for group in stage {
+                println!("\t\tseq![");
+                for system in group {
+                    let system: &SystemId = system;
+
+                    let mut name = map
+                        .iter()
+                        .find(|&(_, id)| *id == *system)
+                        .map(|(name, _)| name)
+                        .unwrap()
+                        .to_string();
```

Comment:
This works, but I would construct new hash map that has the inverse mapping before the loop.

Reply:
Yeah I know, but then it's just a one-time print that doesn't have to be super optimized.

Reply:
I just think it might be clearer that way.